### DISCUSSION TITLE
Music visualization outplug plugin for MPD

### DISCUSSION THREAD

**Discussion by jcorporation** [jcorporation/myMPD#695] [Ideas]
Looks interesting: https://github.com/MusicPlayerDaemon/MPD/pull/1449

**Comment by maxycatz** [comment]
Did this go anywhere?

With scripting could an audio visualiser be an option within myMPD?

**Reply by jcorporation** [reply]
I know no status about this.

This is nothing that can be implemented with the lua scripting option for myMPD.
